as/cNFT/assembly/models/persistent_tokens.ts

```diff
@@ -1,12 +1,14 @@
 import { PersistentSet, PersistentUnorderedMap, u128 } from 'near-sdk-as'
 import { AccountId, TokenId } from '../types'
+import { TokenMetadata } from './persistent_tokens_metadata'
 
 @nearBindgen
 export class Token {
     id: string
     owner_id: string
     creator_id: string
     prev_owner_id: string
+    metadata: TokenMetadata
```

Comment:
Rest looks good. I was wondering why you added metadata here?

Reply:
Because `Token` is used here as a return type, I have to add it

https://github.com/curaOS/contracts/blob/3c8e3687c07a699fa1fdb72626acf1ffc8c0d35e/as/cNFT/assembly/core.ts#L7-L16

Reply:
I see, could you add a `TODO` comment on top of `export class Token {` to improve it :)

Reply:
Done